config/taxonomy/example/module/capability.yaml

```diff
@@ -3,8 +3,6 @@ definitions:
     properties:
       capability:
         enum: [copy, write, read, transform]
-      scope:
-        enum: [asset, workload, cluster]
```

Comment:
Why was this removed?

Reply:
@shlomitk1 how is scope used?
Keep in mind that the taxonomy target cross components definitions., where is scope used elsewhere?

Reply:
cross component + configurable at runtime